File: 백준/Bronze/1978. 소수 찾기/소수 찾기.swift
// [Int] N (immutable): 주어진 수의 개수 (범위: <= N)
let N: Int = Int(readLine() ?? "1") ?? 1

// [Int] countOfPrime (mutable): 소수의 개수를 세는 변수
var countOfPrime: Int = 0

// [Array<Int>] numberList (mutable): 검사할 숫자들을 담는 리스트 (범위: <= 1,000, 길이: N)
var numberList: Array<Int> = (readLine() ?? "1").split(separator: " ").map { Int($0) ?? 1}

// [func(Int)->Bool] isPrime: 소수인지 판별하는 함수
func isPrime(_ num: Int) -> Bool {
    // 1 이하는 소수가 아님
    if (num < 2) { return false }
    
    // 2부터 num-1까지 수 중에서 나누어 떨어지는 수가 있으면 소수가 아님.
    for i in 2..<num {
        if num % i == 0 {
            return false
        }
    }
    return true
}

for number in numberList {
    if (isPrime(number)) {
        countOfPrime+=1
    }
}
            
// 소수의 개수를 출력한다.
print(countOfPrime)

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
# [int] N: 주어진 수의 개수 (범위: <= N)
N = int(input())

# [int] countOfPrime: 소수의 개수를 세는 변수
countOfPrime = 0

# [list(int)] numberList: 검사할 숫자들을 담는 리스트 (범위: <= 1,000, 길이: N)
numberList = list(map(int, input().split()))

for number in numberList:
    # 2 이하는 소수가 아님.
    if (number < 2):
        continue

    # 1과 자신을 제외한 수로 나눠지면 소수가 아님
    for n in range(2, number):
        if (number % n == 0):
            break

    # 끝까지 통과하면 countOfPrime에 1을 더한다.
    else:
        countOfPrime += 1
            
# 소수의 개수를 출력한다.
print(countOfPrime)